terraform/modules/apps/content-store/main.tf

```diff
@@ -57,7 +57,7 @@ module "app" {
         { "name" : "GOVUK_STATSD_PREFIX", "value" : "fargate" },
         { "name" : "GOVUK_USER", "value" : "deploy" }, # TODO: clean up?
         { "name" : "GOVUK_WEBSITE_ROOT", "value" : var.govuk_website_root },
-        { "name" : "MONGODB_URI", "value" : "mongodb://${var.mongodb_host}/content_store_production" },
+        { "name" : "MONGODB_URI", "value" : "${var.draft ? "mongodb://${var.mongodb_host}/draft_content_store_production" : "mongodb://${var.mongodb_host}/content_store_production"}" },
```

Comment:
This should just be passed in from the caller. We should be passing in the URI rather than just the hostname.